server.js

```diff
@@ -1,8 +1,9 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import listEndpoints from "express-list-endpoints"
```

Comment:
I also use this express-list-endpoints but to me it doesn't show the happy thoughts but the endpoints itself instead.